pyproject.toml

```diff
@@ -367,9 +373,10 @@ no_dot = true
 show_deps = true
 
 [tool.codespell]
-# local codespell matches `./docs`, pre-commit codespell matches `docs`
-skip = "*.lock,.direnv,.git,./docs/_freeze,docs/_freeze/**"
-ignore-regex = '\b(i[if]f|I[IF]F|AFE)\b'
+# notebooks are skipped because there's no straightforward way to ignore base64
+# encoded strings
+skip = "*.lock,.direnv,.git,docs/_freeze/**/html.json"
+ignore-regex = '\b(DOUB|i[if]f|I[IF]F|lamduh|AFE)\b'
```

Comment:
These lines look a bit old. Do you need to rebase on top of `master`?

Reply:
Thanks for pointing that out, I think I screwed this one up while resolving conflicts during the previous rebase.